server/src/providers/transactions.ts

```diff
@@ -0,0 +1,100 @@
+import TransactionsModel from '../models/transactions'
+import logger from '../helpers/pino-logger'
+
+const log = logger.child({ from: 'Transactions' })
+
+type transaction = {
+  hash: string,
+  value: number,
+  blockNumber: string,
+  time: number,
+  from: string,
+  date: string,
+  to: string,
+}
+
+class Transactions {
```

Comment:
we dont need to save transactions, we have the blockchain for that.
so most of this is not required
we save only aggregated data
when going over Transfer events
1. sum total amount per day
2. count num txes per day
then $inc in database each day totals
then $inc in database global total amount + total txs

3. add methods to get averages for last 10 days,30 days,90 days